packages/frontend/pages/records/[slug].tsx

```diff
@@ -6,16 +6,11 @@ interface RecordProps {
 }
 
 const RecordPage: NextPage<RecordProps> = ({ record }) => {
-  console.log("record", record);
-  // Get an array of subcategory names
-  // @ts-ignore
-  const subCategoryNames = record.subcategories?.map(subCategoryOnRecord => subCategoryOnRecord.subCategory.name);
-
   return (
     <div className="mx-auto max-w-xl">
       <h1 className="font-bold text-3xl mb">{record.title}</h1>
       <span className="mb-8 block">posted on {record.date}</span>
-      <p className="mb-2">{record.summary}</p>
+      <div className="mb-2" dangerouslySetInnerHTML={{ __html: record.content }}></div>
```

Comment:
We should be careful about it. We need to clean the HTML before saving it.

Reply:
Yep, we will clean it up before importing.